eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -8,6 +8,9 @@ parameters:
   - name: ServiceDirectory
     type: string
     default: not-specified
+  - name: TestProxy
```

Comment:
down below we call into `ci.yml`. We need to add additional parameter so the call looks like this:

```
      - template: /eng/pipelines/templates/jobs/ci.yml
        parameters:
          ServiceDirectory: ${{ parameters.ServiceDirectory }}
          TestProxy: ${{ parameterse.TestProxy }}
          Artifacts: ${{ parameters.Artifacts }}
```

Reply:
Parameters are `compile-time` constants. So technically it'll _work_ if we miss adding it to each level of templates, but it'll be super confusing to debug. Better to pass all the way through.

Reply:
IN `jobs/ci.yml`, we need to add the parameter (and then pass it to ci.tests.yml -> tests.yml)

Reply:
Does it look good now?

Reply:
Yes! At least, I think so :)